action.yaml

```diff
@@ -30,10 +35,12 @@ inputs:
     required: false
     type: integer
 outputs:
-  created_pages:
+  urls_with_actions:
```

Comment:
How about `url_changes` instead?

Reply:
Hmm, I'm not pleased with `urls_with_actions `. The thing is that `url_changes` could imply that only the urls where some changes have been made are returned. Instead, the action returns a list of all URLs to topics in a map with the action that was taken, which could be `noop` or `skip` which means that no changes were made. `create` and `update` are also included when changes have been made. Maybe `url_action_map`?